contrib/verify-commits/verify-commits.py

```
@@ -150,7 +150,20 @@ def main():
+            # This merge-tree functionality requires git >= 2.38. The
+            # --write-tree option was added in order to opt-in to the new
+            # behavior. Older versions of git will not recognize the option and
+            # will instead exit with code 128.
+            try:
+                recreated_tree = subprocess.check_output([GIT, "merge-tree", "--write-tree", parents[0], parents[1]]).decode('utf8').splitlines()[0]
+            except subprocess.CalledProcessError as e:
+                if e.returncode == 128:
+                    print("git v2.38+ is required for this functionality.", file=sys.stderr)
+                    sys.exit(1)
+                else:
+                    raise e
```

Ruff (0.14.5)

Use `raise` without specifying exception name

Remove exception name

(TRY201)